tests/integration/conftest.py

```diff
@@ -288,3 +289,67 @@ def factory(path: Path) -> ConfigUtils:
         return ConfigUtils(path)
 
     return factory
+
+
+@pytest.fixture
+def get_system_release(shell):
```

Comment:
```suggestion
def system_release(shell):
```
This fixture is used as a function parameter or a variable. This name is misleading in a way that it leads people thinking that it's a function.

```python
if "centos-7" in get_system_release:
```
The above looks like a mistake of the programmer - the name implies it is a function which should be called. Variables are not typically named _"do_this"_.

The same goes for the `install_required_packages` fixture bellow which I'd recommend renaming to `required_packages`.

Reply:
Sorry for the out of scope comment. I thought that you added the fixture in this PR but it's been there for quite some time. Anyway, thanks for changing its name.